packages/kbn-esql-validation-autocomplete/src/shared/helpers.ts

```diff
@@ -860,7 +864,8 @@ export function getExpressionType(
         const param = getParamAtPosition(signature, i);
         return (
           param &&
-          (param.type === argType ||
+          (param.type === 'any' ||
```

Comment:
This is a small bug-fix. Soon, probably in https://github.com/elastic/kibana/issues/188978, we'll abandon `any` in operator and function definitions, but it's there for now.